builders/build/substrate-api/polkadot-js-api.md

```diff
@@ -0,0 +1,298 @@
+---
+title: Polkadot.js API
+description: Follow this tutorial to learn the basic of how to use the polkadot.js API library to interact with Moonbeam networks.
+---
+# Polkadot.js API Library
+
+![Intro diagram](/images/builders/build/substrate-api/polkadot-js-api/polkadot-js-api-banner.png)
```

Comment:
The official Polkadot.js documentation is still using the old orange logo. Should we wait until that's updated to the new one to change? Might cause some confusion otherwise. https://polkadot.js.org/docs/api/

Reply:
Replaced with the new logo.